src/App.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from 'react';
+
+function App() {
+  return (
+    <main>
+      <NoughtsCrosses />
+    </main>
+  );
+}
+
+function NoughtsCrosses() {
+  const defaultGame = {
+    board: Array(9).fill(null),
+    isXNext: true,
+    history: [Array(9).fill(null)],
+  };
+  const [game, setGame] = useState(defaultGame);
```

Comment:
Using `useState` here definitely works. This component is easy to read and follow what's happening. One potential advantage to using `useReducer` is your reducer function could store a lot of the logic around checking for the winner and updating the board and the history. In this case, this component didn't get too big so I think this works well as-is.